apps/examples/usb_mouse/mouse_main.c

```diff
@@ -0,0 +1,160 @@
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+
+#include <sys/types.h>
+#include <sys/boardctl.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <debug.h>
+
+#include <tinyara/input/mouse.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+#define MOUSE_DEVPATH "/dev/mouse0"
+
+/****************************************************************************
+ * Name: mouse_main
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+int main(int argc, FAR char *argv[])
+#else
+int mouse_main(int argc, char *argv[])
+#endif
+{
+	struct mouse_report_s sample;
+	ssize_t nbytes;
+	int fd;
+	int errval = 0;
+
+	/* If this example is configured as an NX add-on, then limit the number of
+    * samples that we collect before returning.  Otherwise, we never return
+    */
+
+	/* Open the touchscreen device for reading */
+
+	printf("mouse_main: Opening %s\n", MOUSE_DEVPATH);
+	fd = open(MOUSE_DEVPATH, O_RDONLY);
+	if (fd < 0) {
+		printf("mouse_main: open %s failed: %d\n",
+			   MOUSE_DEVPATH, errno);
+		errval = 2;
+		goto errout_with_tc;
+	}
+
+	/* Now loop the appropriate number of times, displaying the collected
+    * touchscreen samples.
+    */
+
+	for (;;) {
+		/* Flush any output before the loop entered or from the previous pass
+      * through the loop.
+      */
+
+		fflush(stdout);
+
+		/* Read one sample */
+
+		printf("Reading...\n");
+		nbytes = read(fd, &sample, sizeof(struct mouse_report_s));
+		printf("Bytes read: %d\n", nbytes);
+
+		/* Handle unexpected return values */
+
+		if (nbytes < 0) {
+			errval = errno;
+			if (errval != EINTR) {
+				printf("mouse_main: read %s failed: %d\n",
+					   MOUSE_DEVPATH, errval);
+				errval = 3;
+				goto errout_with_dev;
+			}
+
+			printf("mouse_main: Interrupted read...\n");
+		} else if (nbytes != sizeof(struct mouse_report_s)) {
+			printf("mouse_main: Unexpected read size=%d, expected=%d, Ignoring\n",
+				   nbytes, sizeof(struct mouse_report_s));
+		}
+
+		/* Print the sample data on successful return */
```

Comment:
Please put comment under else & make it like:
} else {
/*  comment  */
...
...
}